docs/UserGuide.md

```diff
@@ -758,11 +797,63 @@ Exit the application.
 
 ### Export Data
 
-<!-- TODO: fill this in -->
+> `export {file_path} [/overwrite]`
+
+Export all modules progress to a JSON-format file. 
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress to
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR 
+    file location]/data`)
+  - Must not coincide with Le Tracker's current tracker file path. (:exclamation:The default tracker file path is `
+    [JAR file location]/data/letracker.json`)
+  - If `/overwrite` is not specified, the file specified in `file_path` must not exist
+- <span style="color:#e46c0a">`/overwrite`</span> : If specified, Le Tracker will overwrite all data in 
+  the file specified in `file_path` with the current modules progress in Le Tracker
+  - If the file specified in `file_path` exists, it must allow reading and writing operations
```

Comment:
This seems like something that should be specified under `file_path`. The user might not have write permissions to a a file existent or non-existent depending on the write permissions of the directory. 

So perhaps it should be `The user must have write permissions to file_path`.

Also, why does the user need read permissions for the file? Isn't write permission enough?

Reply:
It's incorrect. Write permission is enough. Sorry